realm/realm-library/src/main/java/io/realm/internal/util/Pair.java

```diff
@@ -27,8 +27,8 @@
  * objects.
  */
 public class Pair<F, S> {
-    public final F first;
-    public final S second;
+    public F first;
```

Comment:
I'm changing it here: https://github.com/realm/realm-java/pull/4415/files#diff-0bc822b1fa3365c424a52d324b4f0bd4R156